solutions/security/detect-and-alert/mitre-attandckr-coverage.md

```diff
@@ -20,7 +20,7 @@ Mirroring the MITRE ATT&CK® framework, columns represent major tactics, and cel
 To access the **MITRE ATT&CK® coverage** page, find **Detection rules (SIEM)** in the navigation menu or look for “Detection rules (SIEM)” using the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md), then go to **MITRE ATT&CK® coverage**.
 
 ::::{note}
-This page only includes the detection rules you currently have installed, and only rules that are mapped to MITRE ATT&CK®. The coverage page maps detections to the following [MITRE ATT&CK® version](https://attack.mitre.org/resources/updates/updates-april-2024) used by {{elastic-sec}}: `v16.1`. Elastic prebuilt rules that aren’t installed and custom rules that are either unmapped or mapped to a deprecated tactic or technique will not appear on the coverage map.
+This page only includes the detection rules you currently have installed, and only rules that are mapped to MITRE ATT&CK®. The coverage page maps detections to the following [MITRE ATT&CK® version](https://attack.mitre.org/resources/updates/updates-april-2025) used by {{elastic-sec}}: `v17.1`. Elastic prebuilt rules that aren’t installed and custom rules that are either unmapped or mapped to a deprecated tactic or technique will not appear on the coverage map.
```

Comment:
With our [new guidelines](https://elastic.github.io/docs-builder/contribute/cumulative-docs/) for writing cumulative docs for 9.0+ and Serverless, we actually need to keep the "old" docs and add to them instead. I was thinking of creating a table to keep the the versioning info contained and organized. I'll throw something together tomorrow and share an example of what I'm considering.

Reply:
Thanks @nastasha-solomon!

Reply:
Hey again! Here's what I'm thinking:

<img width="670" height="1164" alt="Screenshot 2025-08-12 at 7 10 38 PM" src="https://github.com/user-attachments/assets/c964a0e0-5f5d-42ca-a922-c51877fcb012" />

As you can see, I've moved the mapping info to a table under the image and included a note that Serverless always uses the latest mapped MITRE ATT&CK® version. If you're happy with these changes, I can push them to the PR.

Reply:
Added in [eed605a](https://github.com/elastic/docs-content/pull/2518/commits/eed605a2e95ce4ef9c65575ecff5eec11eb4542e).